pkgs/os-specific/linux/purism/librem-ec-acpi/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kernel,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librem-ec-acpi";
+  version = "0.9.2-${kernel.version}";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "nicole.faerber";
+    repo = "librem-ec-acpi-dkms";
+    rev = "cec8c0e8bf1532c9c605f60bb02a2ef0f98e5d77";
```

Comment:
Nit: this is the commit hash for the tag, but it'd be clearer to just reference the tag name in my opinion. That'd probably mean you'd want something like

```nix
let
  version = "0.9.2";
in
stdenv.mkDerivation {
  version = "${version}-${kernel.version}";
  src = fetchFromGitLab {
    rev = "v${version}";
    ...
  };
  ...
}
```